backend/src/akvo/lumen/endpoint/aggregation.clj

```diff
@@ -4,6 +4,9 @@
             [akvo.lumen.lib.aggregation :as aggregation]
             [cheshire.core :as json]
             [compojure.core :refer :all]
+            [akvo.lumen.specs.components :refer (integrant-key)]
```

Comment:
Since we used vectors most of the places, is there a reason for using the list notation?